### PR TITLE
lwimiq

### PR DESCRIPTION
**Pull request by jntullo**
Last week in MIQ!

### CODE REVIEW

File: site/_posts/2016-10-24-lwimiq-exceptions-logs-ternary.html.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+---
+title: "Last Week in ManageIQ: Exceptions, Logs, and Ternary Operators!"
+author: jntullo
+date: 2016-10-24
+comments: true
+published: true
+tags: LWIMIQ
+---
+
+Hello out there, ManageIQ Community. [Jillian](https://github.com/jntullo) here again with the latest edition of Last Week in Manage IQ!
```
Comment (imtayadeway) [inline site/_posts/2016-10-24-lwimiq-exceptions-logs-ternary.html.md]: s/Manage IQ/ManageIQ/ (I /think/)

**Comment by imtayadeway** [comment]
:+1: LGTM other than couple of comments above. :balloon:

File: site/_posts/2016-10-24-lwimiq-exceptions-logs-ternary.html.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+### [Remove Unnecessary Ternary Operators](https://github.com/ManageIQ/manageiq/pull/12012)
+[Roman](https://github.com/romanblanco) improved toolbar builder by deleting some unnecessary ternary operators.
+
+## Wrapping up
+Thanks for joining us for another edition of LWIMIQ. See you next week!
+
+[open issues]: https://github.com/ManageIQ/manageiq/issues
+[contributors]: https://github.com/ManageIQ/manageiq/graphs/contributors
+[PRs merged]: https://github.com/ManageIQ/manageiq/pulls?page=1&q=is%3Apr+is%3Amerged+base%3Amaster+merged%3A%222016-10-17+..+2016-10-24%22+sort%3Acreated-desc&utf8=%E2%9C%93
+[commits]: https://github.com/manageiq/manageiq/compare/master@%7B2016-10-17%7D...@%7B2016-10-24%7D
```
Comment (jntullo) [inline site/_posts/2016-10-24-lwimiq-exceptions-logs-ternary.html.md]: I get that wrong every time 😛

**Comment by JPrause** [comment]
Thanks @jntullo for the submission.
@imtayadeway thanks for reviewing so quickly.
@juliancheal and @ZitaNemeckova can you also give this a quick review.

**Comment by ZitaNemeckova** [comment]
Seems nice 👍

**Comment by juliancheal** [comment]
LGTM 👍 Great job @jntullo.

**Comment by JPrause** [comment]
Thanks @ZitaNemeckova  and @juliancheal for reviewing!

@hayesr here's the latest blog entry. Merge away.

**Comment by syncrou** [comment]
@jntullo - A little late to the 🎉  - Looks great 👍

**Comment by JPrause** [comment]
@syncrou not late at all. Thanks as always for your help!

**Pull request merged**